crates/bevy_text/src/draw.rs

```diff
@@ -119,15 +119,10 @@ impl<'a> Drawable for DrawableText<'a> {
                     )?;
 
                     let bounds = outlined.px_bounds();
-                    let offset = scaled_font.descent() + glyph_height;
-                    let transform = Mat4::from_translation(
-                        caret
-                            + Vec3::new(
-                                0.0 + glyph_width / 2.0 + bounds.min.x,
-                                glyph_height / 2.0 - bounds.min.y - offset,
-                                0.0,
-                            ),
-                    );
+                    let x = bounds.min.x + glyph_width / 2.0;
+                    // the 0.5 accounts for odd-numbered heights (bump up by 1 pixel)
+                    let y = -bounds.max.y + glyph_height / 2.0 + 0.5;
```

Comment:
```suggestion
                    let y = -bounds.max.y + glyph_height / 2.0 - scaled_font.descent() + 0.5;
```

Reply:
This fixed the offset for me

Reply:
I think I'll just merge this in so this fix can be included in Bevy 0.3

Reply:
Oh, of course. My initial PR commit was closer to the top edge (which is where the images came from), but I realised I had an incorrect assumption about the offset so I fixed it, without realising I didn't account for the offset from the baseline.